lib/splunk-sdk-ruby/version.rb

```diff
@@ -23,5 +23,5 @@
 # release new versions.
 #
 module Splunk
-  VERSION = '1.0.3'
+  VERSION = '1.0.4.1'
```

Comment:
I would remove this change - no need to modify the version - we'll handle it during our release.